Lib/extractor/formats/opentype.py

```diff
@@ -299,12 +299,14 @@ def _extractInfoGasp(source, info):
 # Tools
 
 def binaryToIntList(value, start=0):
-    string = num2binary(value)
-    all = []
-    for index, v in enumerate(reversed(string)):
-        if v == "1":
-            all.append(start + index)
-    return all
+    intList = []
+    counter = 0
```

Comment:
you could add the `start` value in here, when you define `counter = start`

Reply:
changed